packages/cli/package.json

```diff
@@ -53,6 +53,9 @@
     "zod": "^3.22.2"
   },
   "peerDependencies": {
+    "@aws-sdk/client-amplify": "^3.465.0",
```

Comment:
These should be runtime deps of this package since it is the package that actually constructs the client provider